pyproject.toml

```diff
@@ -159,6 +159,9 @@ lint.select = [
     "SLF",     # self - https://docs.astral.sh/ruff/settings/#lintflake8-self
     "PLC2701", # private import - https://docs.astral.sh/ruff/rules/import-private-name/
 ]
+lint.ignore = [
```

Comment:
Change to this file might be better in a separate PR

Reply:
#704